### PR TITLE
Add onChangeHeight function

### PR DESCRIPTION
**Pull request by miraheka2305**

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -1,18 +0,0 @@
-{
```
Comment (shuLhan) [inline tslint.json]: This file should not be deleted nor renamed to "tslint 2.json`.  Any reason to do so?

Comment (miraheka2305) [reply tslint.json]: i think i haven't deleted nor renamed that file, but i've ```rm -rf node_modules``` and ```npm i``` again. Should i renamed the file like before ?

Comment (shuLhan) [reply tslint.json]: Yes.

**Review by shuLhan — CHANGES_REQUESTED** [review]
Please do a git rebase first. Many changes delete lines from previous commits.

**Pull request closed without merging**